src/Umbraco.Web.UI.Client/src/views/datatypes/delete.html

```diff
@@ -92,14 +92,14 @@ <h5 class="mt4" style="margin-bottom: 20px;">
                             <table class="table table-condensed table-bordered">
                                     <thead>
                                         <tr>
-                                            <th><localize key="general_name">Name</localize></th>
+                                            <th class="umb-dialog-datatype-delete__table-head-column-name"><localize key="general_name">Name</localize></th>
                                             <th><localize key="general_properties">Properties</localize></th>
                                         </tr>
                                     </thead>
                                     <tbody>
                                         <tr ng-repeat="relation in vm.references.memberTypes">
                                             <td><span title="{{::relation.name}}({{::relation.alias}})"><i class="umb-table-body__icon {{relation.icon}}"></i>{{::relation.name}}</span></td>
-                                            <td><span><span class="red" ng-repeat="property in relation.properties"><i class="icon icon-alert red"></i>{{::property.name}}{{$last ? '' : ', '}}</span></span></td>
+                                            <td><span><p class="red" ng-repeat="property in relation.properties"><i class="icon icon-alert red"></i>{{::property.name}}</p></span></td>
```

Comment:
Is this semantically correct HTML to have a block element inside an inline element @nielslyngsoe ?

Reply:
No, its wrong, so have changed it now.